src/main/scala/loamstream/loam/intake/IntakeSyntax.scala

```diff
@@ -17,11 +17,16 @@ import loamstream.util.Loggable
 object IntakeSyntax extends IntakeSyntax
 
 trait IntakeSyntax extends Interpolators {
+  type ColumnDef = loamstream.loam.intake.ColumnDef
+  val ColumnDef = loamstream.loam.intake.ColumnDef
```

Comment:
Can you explain this change? Going from type alias to type constructor?

Reply:
I suspect I wanted to be able to say `import IntakeSyntax._` and get "everything" including types/objects from `loamstream.loam.intake` that would otherwise need to be imported manually.